packages/fluent/scss/popover/_variables.scss

```diff
@@ -0,0 +1,89 @@
+/// Font size of the popover.
+/// @group popover
+$kendo-popover-font-size: $kendo-window-font-size !default;
+/// Font family of the popover.
+/// @group popover
+$kendo-popover-font-family: $kendo-window-font-family !default;
+/// Line height of the popover.
+/// @group popover
+$kendo-popover-line-height: $kendo-window-line-height !default;
+
+/// Background color of the popover.
+/// @group popover
+$kendo-popover-bg: var( --kendo-component-bg, initial ) !default;
+/// Text color of the popover.
+/// @group popover
+$kendo-popover-text:  var( --kendo-component-text, initial ) !default;
+/// Border color of the popover.
+/// @group popover
+$kendo-popover-border:  var( --kendo-component-border, initial ) !default;
+/// Shadow of the popover.
+/// @group popover
+$kendo-popover-shadow: $kendo-popup-shadow !default;
+
+/// Horizontal padding of the popover header.
+/// @group popover
+$kendo-popover-header-padding-x: $kendo-dialog-titlebar-padding-x !default;
+/// Vertical padding of the popover header.
+/// @group popover
+$kendo-popover-header-padding-y: $kendo-dialog-titlebar-padding-y !default;
+/// Border width of the popover header.
+/// @group popover
+$kendo-popover-header-border-width: $kendo-dialog-titlebar-border-width !default;
+/// Border style of the popover header.
+/// @group popover
+$kendo-popover-header-border-style: $kendo-popover-border-style !default;
+/// Background color of the popover header.
+/// @group popover
+$kendo-popover-header-bg: $kendo-dialog-titlebar-bg !default;
+/// Text color of the popover header.
+/// @group popover
+$kendo-popover-header-text: $kendo-dialog-titlebar-text !default;
+/// Border color of the popover header.
+/// @group popover
+$kendo-popover-header-border: $kendo-dialog-titlebar-border !default;
+
+/// Horizontal padding of the popover body.
+/// @group popover
+$kendo-popover-body-padding-x: $kendo-window-inner-padding-x !default;
+/// Vertical padding of the popover body.
+/// @group popover
+$kendo-popover-body-padding-y: $kendo-card-body-padding-y !default;
+
+/// Border width of the popover actions.
+/// @group popover
+$kendo-popover-actions-border-width: 0 !default;
+
+/// Width of the popover callout.
+/// @group popover
+$kendo-popover-callout-width: $kendo-card-callout-size !default;
+/// Height of the popover callout.
+/// @group popover
+$kendo-popover-callout-height: $kendo-card-callout-size !default;
+/// Border width of the popover callout.
+/// @group popover
+$kendo-popover-callout-border-width: $kendo-popover-border-width !default;
+/// Border style of the popover callout.
+/// @group popover
+$kendo-popover-callout-border-style: $kendo-popover-border-style !default;
+/// Background color of the popover callout.
+/// @group popover
+$kendo-popover-callout-bg: $kendo-popover-bg !default;
+/// Border color of the popover callout.
+/// @group popover
+$kendo-popover-callout-border: $kendo-popover-border !default;
```

Comment:
do we need the $kendo-popover-callout-bg and $kendo-popover-callout-border variables? We don't use them anywhere

Reply:
I have remove unnecessary variables.